### DISCUSSION TITLE
Need for R code?

### DISCUSSION THREAD

**Discussion by PMassicotte** [cloudnativegeo/cloud-optimized-geospatial-formats-guide#83] [Ideas]
Could be interesting to use tabsets (https://quarto.org/docs/output-formats/html-basics.html#tabsets) to provide different code approaches.

If interested, I could contribute to that.

**Comment by kylebarron** [comment]
I think it would be great to have R code included as well. Unfortunately, I don't know R code well enough to review PRs, but @wildintellect likely does

**Reply by wildintellect** [reply]
Oh yes I can review and R based code @PMassicotte see also https://github.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/issues/91

**Reply by wildintellect** [reply]
tagging @cboettig as another potential participant

**Reply by wildintellect** [reply]
FYI we're working (NASA MAAP& VEDA projects) on some basic code for a few formats (sf and Terra supported) and might be looking for reviewers in the next couple of weeks. cc: @PMassicotte

**Reply by PMassicotte** [reply]
Very cool! I work a lot with that kind of data. Happy to help!

**Comment by wildintellect** [comment]
I looked into this a little more, and I'm not sure it will work, as the syntax has to be markdown. I'm going to test making a markdown block in ipynb and see if quarto picks it up but that would still mean writing the code in markdown not notebook cells so it wouldn't be executable.

**Reply by wildintellect** [reply]
https://quarto.org/docs/authoring/notebook-embed.html might be a way (seems to do outputs not code-blocks) following https://github.com/quarto-dev/quarto-cli/discussions/4232
